### ISSUE TITLE
Fetching data from the Argo reference dataset 

### ISSUE THREAD

**Issue by kamwal** [euroargodev/argodmqc_owc#53]
It's probably still too early, but argopy is able to fetch and return data from the Argo reference dataset !

```python
from argopy import DataFetcher as ArgoDataFetcher
loader = ArgoDataFetcher(src='erddap', ds='ref')
ds1 = loader.region([-85, -45, 10, 20, 0, 1000, ‘2012-01’, ‘2012-02’]).to_xarray()
```
we could develop an access point that would work with owc, ie using distances instead of a rectangular space/time domain

_Originally posted by @gmaze in https://github.com/euroargodev/argodmqc_owc/issues/30#issuecomment-730339090_

**Comment by imab4bsh** [comment]
Dear @kamwal and @gmaze,
Do we currently have API access to the CTD reference database via pyowc? 
What I understand is that argopy can access the Argo reference database, but the CTD reference data used in pyowc is currently accessed locally reading the mat files.

**Comment by kamwal** [comment]
Currently, there is no available API to access CTD reference data via pyowc. CTD data can be only accessed by pyowc locally by reading your mat files from your local archive. 
@gmaze please correct me if I am wrong and something has changed.

**Comment by gmaze** [comment]
@kamwal you're right, CTD reference data are not yet available from argopy since it's not available from the Ifremer erdapp either.  Only Argo reference data are available from argopy

**Comment by imab4bsh** [comment]
Ok, thanks. And which format shoud the CTD reference have to make it available in the Ifremer erdapp? netCDF I guess... It is possible to password protect the data in the erdapp?

**Comment by gmaze** [comment]
The issue is not the format but the user authentification
It's possible to have an erddap behind a login/password, so that CTD ref data are made available like today with the ftp and mat files.
But this has to be implemented by the Coriolis team ...

**Comment by gmaze** [comment]
> I think this ticket is not relevant with a main code functionality

sorry @kamwal I don't understand

**Comment by gmaze** [comment]
hi there,
I forget to mention that argopy is now able to retrieve both the Argo and the ship-CTD reference dataset !

## Argo floats reference dataset for DMQC
```python
from argopy import DataFetcher
loader = DataFetcher(src='erddap', ds='ref')
ds = loader.region([-85, -45, 10, 20, 0, 1000, ‘2012-01’, ‘2012-02’]).to_xarray()
```

## Ship-CTD reference dataset for DMQC
Using user/password new argopy options, it is possible to fetch the [Argo CTD reference database](http://www.argodatamgt.org/DMQC/Reference-data-base/Latest-Argo-Reference-DB), with the [CTDRefDataFetcher](https://argopy.readthedocs.io/en/latest/generated/argopy.CTDRefDataFetcher.html#argopy.CTDRefDataFetcher) class. ([#256](https://github.com/euroargodev/argopy/pull/256)) by [G. Maze](http://www.github.com/gmaze)

```python
from argopy import CTDRefDataFetcher

with argopy.set_options(user="john_doe", password="***"):
    f = CTDRefDataFetcher(box=[15, 30, -70, -60, 0, 5000.0])
    ds = f.to_xarray()
```

We're waiting for someone at @euroargodev/coriolis  to update the erddap server for you to be able to use your login/password as provided by  @coatanoan  at the time of your request of the reference dataset.

When this will be done, we'll be good to go to update pywoc to fetch reference data online
